src/components/Notifications/Notifications.tsx

```diff
@@ -497,7 +497,7 @@ export class TurnIndicator extends React.Component<{}, any> { /* {{{ */
     render() {
         return (
             <span className="turn-indicator" onClick={this.advanceToNextBoard}>
-                <span className={this.state.total > 0 ? "active count" : "count"}><span>{this.state.count}</span></span>
+                <span className={this.state.total > 0 ? (this.state.count > 0 ? "active count" : "inactive count") : "count"}><span>{this.state.count}</span></span>
```

Comment:
I wasn't aware you could add an extra condition to this statement...

is this how it works?
```typescript
this.state.a ? this.state.b ? b.true : a.true : both.false
```
or something else?

Reply:
Be aware of the parenthesis. It's simply a `_?_:_` nested into another `_?_:_`.

Reply:
Oh lol thanks I didn't see the parenthesis under the highlighting :rofl: